src/Microsoft.IdentityModel.Tokens/Encryption/SymmetricKeyWrapProvider.cs

```diff
@@ -371,7 +371,7 @@ public override byte[] WrapKey(byte[] keyBytes)
                 throw LogHelper.LogArgumentNullException(nameof(keyBytes));
 
             if (keyBytes.Length % 8 != 0)
-                throw LogHelper.LogExceptionMessage(new ArgumentException(LogHelper.FormatInvariant(LogMessages.IDX10664, keyBytes.Length << 3), nameof(keyBytes)));
+                throw LogHelper.LogExceptionMessage(new ArgumentException(LogHelper.FormatInvariant(LogMessages.IDX10664, LogHelper.MarkAsNonPII(keyBytes.Length << 3)), nameof(keyBytes)));
```

Comment:
LogHelper.MarkAsNonPII(nameof(keyBytes)) ?

Reply:
nameof(keyBytes) falls outside Format.Invariant and cannot be marked as NonPII